docs/wiki/configurator/motors-tab.mdx

```diff
@@ -85,6 +85,14 @@ out of dives and decreased to avoid unwanted downward thrust during inverted man
 position and the lowest stick position now produces maximum negative throttle, with maximum positive throttle at the 
 highest stick position
 
+:::info
+
+Air mode does not work properly when using 3D mode, you should disable permanent Air Mode in the 
+[Configuration Tab](https://betaflight.com/docs/wiki/configurator/configuration-tab#other-features) and configure Air 
+Mode on a switch in the [Modes Tab](https://betaflight.com/docs/wiki/configurator/modes-tab) so that Air Mode is 
+disabled when 3D mode is activated. Simply set the "Air Mode" mode range to be the same switch channel and values as
+ the "Disable 3D Mode" mode range.
+
```

Comment:
Missing closing :::

Airmode feature should be disabled in firmware when using 3D mode.

Reply:
> 

Just spotted that myself, fixed now. Also added warning about GPS RTH